packages/icons/icons.json

```diff
@@ -4198,5 +4198,89 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Two column view",
+        "iconName": "2-column-view",
+        "tags": "presentation, arrangement, layout, two, grid, columns",
+        "group": "interface",
```

Comment:
not sure if "interface" is the most appropriate group here

Reply:
seems fine 👍🏽